src/yunohost/hook.py

```diff
@@ -355,13 +355,13 @@ def hook_exec(path, args=None, raise_on_error=False, no_trace=False,
     command.append(cmd.format(script=cmd_script, args=cmd_args))
 
     if logger.isEnabledFor(log.DEBUG):
-        logger.info(m18n.n('executing_command', command=' '.join(command)))
+        logger.debug(m18n.n('executing_command', command=' '.join(command)))
     else:
-        logger.info(m18n.n('executing_script', script=path))
+        logger.debug(m18n.n('executing_script', script=path))
 
     # Define output callbacks and call command
     callbacks = (
-        lambda l: logger.info(l.rstrip()),
+        lambda l: logger.debug(l.rstrip()),
```

Comment:
@maniackcrudelis Beware of this change : I don't know how that impacts the whole package_check / app CI thing but the whole detail of app installs will now start with 'DEBUG' at the beginning of the line instead of 'INFO' (on stretch). Also `--verbose` won't exist anymore

Reply:
Actually the CI uses --debug, not --verbose and looks for the lines marked WARNING and ERROR.
So, it shouldn't break anything on our CI.
Anyway, our CI stretch should show us if something doesn't work anymore.